docs/user-guide/remapping.ipynb

```diff
@@ -0,0 +1,557 @@
+{
```

Comment:
<p>I'd suggest cutting down the content a bit here and simplifying it to the following:</p><p> </p><p>Remapping, or commonly referred to as Regridding, is the process of taking data that resides on one grid and mapping it to another. Details on various remapping methods can be found here. This user guide section will cover the two native remapping methods that are supported by UXarray:</p><ul><li>Nearest Neighbor</li><li>Inverse Distance Weighted</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/UXARRAY/uxarray/pull/807/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>